### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by ceciliaxxy** [SunLab-UGA/RF-3DGS#5]
Thank you to the authors for kindly sharing the code. I have a few questions:

1. In the second phase, when training the RRF with Gaussians, how are the camera-related parameters determined? The positions and rotations are clear, but the parameters in the file cameras.txt are somewhat unclear to me. I am confused about whether they are related to the number of receiving antennas.
2. The pathloss spectrum is straightforward, but I am unsure about the ToF，phase，DOA spectra. For the pathloss spectrum, the pathloss contribution of each Gaussian is well defined. However, it is unclear how the ToF, phase, and DOA change when passing through Gaussians, as their behavior seems different from that of pathloss.
3.  I first ran train.py with the ModelParams set as:
self._source_path = "./RF-3DGS_dataset/training-rf-spectrum/3dgs_MPC_100"
self._model_path = "./output/rf-3dgs_MPC_test"
Then, I ran render.py using the optimized Gaussian parameters obtained after the training phase. However, the rendered spectra are quite different from the ground truth, and I am not sure why this discrepancy occurs. Please give me some suggestions.

**Comment by ceciliaxxy** [comment]
Additionally, I encountered an issue during training. When running train.py with the following configuration:
source_path: ./RF-3DGS_dataset/training-rf-spectrum/3dgs_MPC_100
model_path: ./output/rf-3dgs_MPC_test
start_checkpoint: RF-3DGS_dataset/blender_visual_trained/chkpnt30000.pth
The training loss remains around 0.3 and doesn't decrease significantly. Is this behavior expected?

**Comment by Lihao-552** [comment]
1. In the second phase, when training the RRF with Gaussians, how are the camera-related parameters determined? The positions and rotations are clear, but the parameters in the file cameras.txt are somewhat unclear to me. I am confused about whether they are related to the number of receiving antennas. 

The "camera" parameters represent the intrinsics of a standard pinhole camera model used specifically for generating the spatial spectrum. While the antenna array itself is configured within the Sionna simulator, your insight is correct: the camera intrinsics must align with the antenna array configurations to maintain energy conservation. This alignment is a specific challenge we are currently addressing in our ongoing work.

2. The pathloss spectrum is straightforward, but I am unsure about the ToF，phase，DOA spectra. For the pathloss spectrum, the pathloss contribution of each Gaussian is well defined. However, it is unclear how the ToF, phase, and DOA change when passing through Gaussians, as their behavior seems different from that of pathloss.

These metrics should not be interpreted at the individual scale of the Gaussians. Instead, they are attributes embedded within the multi-dimensional Radio Radiance Field (RRF) that we have defined. Gaussian Splatting is utilized here simply as a computational tool to approximate two primary functions: the geometric density function and the radio radiance function. For ToF, this orginal radiance field is not accurate, we have update this in our later work RF-PGS which you can find in our github repo. For angle of departure, the behaivour is actually similar to the pathloss.

3. I first ran train.py with the ModelParams set as:
self._source_path = "./RF-3DGS_dataset/training-rf-spectrum/3dgs_MPC_100"
self._model_path = "./output/rf-3dgs_MPC_test"
Then, I ran render.py using the optimized Gaussian parameters obtained after the training phase. However, the rendered spectra are quite different from the ground truth, and I am not sure why this discrepancy occurs. Please give me some suggestions.

It is expected that these would appear quite different. This discrepancy occurs because the training spatial spectra are mapped onto the high-resolution Gaussian geometry during the optimization process.

4. Additionally, I encountered an issue during training. When running train.py with the following configuration:
source_path: ./RF-3DGS_dataset/training-rf-spectrum/3dgs_MPC_100
model_path: ./output/rf-3dgs_MPC_test
start_checkpoint: RF-3DGS_dataset/blender_visual_trained/chkpnt30000.pth
The training loss remains around 0.3 and doesn't decrease significantly. Is this behavior expected?

I am unsure which specific loss metric you are referring to, but if it is the loss displayed in the progress bar, that represents the loss for an individual image. Since we utilize a single image per iteration and the image order is randomized within each epoch, a consistent decrease in loss for each specific training image may not be directly observable in the real-time logs.
